tsconfig.json

```diff
@@ -2,12 +2,13 @@
   "compilerOptions": {
     "moduleResolution": "node",
     "target": "es2015",
-    "noEmit": true,
+    // "noEmit": true,
```

Comment:
comments in json make me sad, if we don't want this option anymore lets just delete it

Reply:
Good call. I'll check

Reply:
Unnecessary. I removed it.